06-building-a-program-start-to-finish/solving-the-problem/output/app.js

```diff
@@ -0,0 +1,131 @@
+angular.module('root', ['ngAnimate', 'toastr'])
+  .controller('invertedController', ($scope, toastr) => {
+    const invertedIndex = new InvertedIndex();
+    $scope.docsMock = [];
+    $scope.indexedFile = {};
+    $scope.fileStore = {};
+    $scope.fileNames = [];
+    $scope.table1 = false;
+    $scope.table2 = false;
+    $scope.searchAll = {};
+
+    $scope.createIndex = () => {
+      $scope.isSearching = false;
+      const getSelectedFile = document.getElementById('uploadedFilesSelect');
+      $scope.uploadedFileName =
+      getSelectedFile.options[getSelectedFile.selectedIndex].text;
+      if ($scope.uploadedFileName === '') {
+        toastr.error('Please select a file and click Create index', 'Error');
+      }
```

Comment:
I believe you need this condition `$scope.uploadedFileName === ''` to be true before lines 20 - 29 are executed. What happens when that condition in not true?